src/components/select/select.js

```diff
@@ -288,19 +288,16 @@ function SelectDirective($mdSelect, $mdUtil, $mdTheming, $mdAria, $compile, $par
             }
           });
 
-        // Wait until postDigest so that we attach after ngModel's
-        // blur listener so we can set untouched.
-        $mdUtil.nextTick(function () {
-          element.on('blur', function() {
-            if (untouched) {
-              untouched = false;
-              ngModelCtrl.$setUntouched();
-            }
 
-            if (selectScope.isOpen) return;
-            containerCtrl && containerCtrl.setFocused(false);
-            inputCheckValue();
-          });
+        element.on('blur', function() {
```

Comment:
Weird, I've checked out your branch, reverted this change, while keeping the rest of the changes, and all tests are passing. Futhermore, I am still not sure how this is going to work, when we are not going to get any blur events without tabindex.